libs/transactions/src/test-setup.ts

```diff
@@ -1 +1,2 @@
 import 'jest-preset-angular/setup-jest';
+import '@angular/localize/init';
```

Comment:
Add a new line at the bottom of this file.

Reply:
fix formatting for all of the project and added format scripts with prettier